lib/active_shipping/carriers/fedex.rb

```diff
@@ -603,18 +607,16 @@ def parse_tracking_response(response, options)
 
 
         first_notification = tracking_details.at('Notification')
-        if first_notification.at('Severity').text == 'ERROR'
-          case first_notification.at('Code').text
-          when '9040'
+        severity = first_notification.at('Severity').text
+        if severity == 'ERROR' || severity == 'FAILURE'
+          message = first_notification.try(:text)
```

Comment:
Can't `try!`, we still support [`active_support 3.2`](https://github.com/Shopify/active_shipping/blob/master/Gemfile.activesupport32), which does not have `try!` 

http://apidock.com/rails/v4.0.2/Object/try%21